In [5]:
import numpy as np

# Constants
current_year = 2024
target_year = 2026
distance_ly = 4.37  # Distance to Alpha Centauri system in light-years

# Proper motion of the whole Alpha Centauri AB system in arcseconds per year
proper_motion_system_arcsec_per_year = 3.7  

# Calculate the time difference
time_difference_years = target_year - current_year

# Calculate the change in position due to proper motion in arcseconds
system_separation_arcsec = proper_motion_system_arcsec_per_year * time_difference_years

# Convert arcseconds to radians
system_separation_radians = np.deg2rad(system_separation_arcsec / 3600)

# Calculate the change in separation in light-years
separation_ly = distance_ly * system_separation_radians

print(f"The change in position of Alpha Centauri AB system in {target_year} is approximately {system_separation_arcsec:.6f} arcseconds.")



The change in position of Alpha Centauri AB system in 2026 is approximately 7.400000 arcseconds.


1. Distance between Alpha Centauri  and earth in light years is 4.37 ly. https://web.archive.org/web/20220223194607id_/http://www.intellectualarchive.com/Journal_Files/IAJ_2018_07_17.pdf

2. Proper motion of Alpha Centauri in arcseconds per year is 3.7 (https://www.aanda.org/articles/aa/full_html/2016/10/aa29201-16/aa29201-16.html)

The above code is solved based on  small-angle approximation where, s≈d×((μ*Δt*π​)/3600*180)

In [13]:
from astroquery.simbad import Simbad


Simbad.add_votable_fields('flux(V)', 'fe_h', 'sp', 'rv_value')

# Query for Alpha Centauri A
result_A = Simbad.query_object("Alpha Cen A")
print(result_A)

# Extracting values if needed
V_mag = result_A['FLUX_V'][0]
print(f"Alpha Cen A: V Magnitude: {V_mag}")

# Spectral Type
spectral_type = result_A['SP_TYPE'][0]
print(f"Alpha Cen A: Spectral Type: {spectral_type}")

# Metallicity
metallicity = result_A['FE_H'][0] if 'FE_H' in result_A.colnames else "Not Available"
print(f"Alpha Cen A: Metallicity [Fe/H]: {metallicity}")

# Radial Velocity
radial_velocity = result_A['RV_VALUE'][0]
print(f"Alpha Cen A Radial Velocity: {radial_velocity} km/s")





  MAIN_ID         RA           DEC      ... RV_VALUE_4 SCRIPT_NUMBER_ID
               "h:m:s"       "d:m:s"    ...   km / s                   
----------- ------------- ------------- ... ---------- ----------------
* alf Cen A 14 39 36.4940 -60 50 02.373 ...    -15.252                1
Alpha Cen A: V Magnitude: 0.009999999776482582
Alpha Cen A: Spectral Type: G2V
Alpha Cen A: Metallicity [Fe/H]: Not Available
Alpha Cen A Radial Velocity: -15.252 km/s


In above code, V magnitude and Effective temperature of Alpha Centauri A and B has been taken from https://adsabs.harvard.edu/full/1986A%26A...158..273S .TOLIMAN bandpass 530-640 nm has been taken from https://doi.org/10.1117/12.3019202

Here, I take Primary mirror diameter (13 cm) and secondary mirror diamater (1.90 cm) and spider_fraction = 0.0214 

Here, I start by modeling Alpha Centauri A's spectral flux density as a blackbody with a temperature of 5781 K and integrating it over the TOLIMAN bandpass (530-640 nm) to determine the total emitted flux. This flux is then normalized using the V-band magnitude (-0.1) and the Vega system zero-point flux to match observed brightness. Next, the effective collecting area of a TOLIMAN telescope is calculated, accounting for losses due to the secondary mirror and spider arm obstructions. The integrated flux is scaled by this effective area, providing an estimate of the light actually collected by the telescope. I also divide the collected flux (in erg/s) by the energy of a single photon (in erg) is the correct approach to convert it into photons per second. 

In [ ]:
from astroquery.simbad import Simbad


Simbad.add_votable_fields('flux(V)', 'fe_h', 'sp', 'rv_value')

# Query for Alpha Centauri B
result_A = Simbad.query_object("Alpha Cen B")
print(result_A)


V_mag = result_A['FLUX_V'][0]
print(f"Alpha Cen B: V Magnitude: {V_mag}")

# Spectral Type
spectral_type = result_A['SP_TYPE'][0]
print(f"Alpha Cen B: Spectral Type: {spectral_type}")

# Metallicity
metallicity = result_A['FE_H'][0] if 'FE_H' in result_A.colnames else "Not Available"
print(f"Metallicity [Fe/H]: {metallicity}")

# Radial Velocity
radial_velocity = result_A['RV_VALUE'][0]
print(f"Alpha Cen B:Radial Velocity: {radial_velocity} km/s")


  MAIN_ID         RA           DEC      ... RV_VALUE_2 SCRIPT_NUMBER_ID
               "h:m:s"       "d:m:s"    ...   km / s                   
----------- ------------- ------------- ... ---------- ----------------
* alf Cen B 14 39 35.0631 -60 50 15.099 ...    -22.586                1
Alpha Cen B: V Magnitude: 1.3300000429153442
Alpha Cen B: Spectral Type: K1V
Metallicity [Fe/H]: Not Available
Alpha Cen B:Radial Velocity: -22.586 km/s


Metallicity   [Fe/H] of alpha centurai  ~ +0.23 dex 
Ref: https://www.aanda.org/articles/aa/full_html/2018/07/aa33125-18/aa33125-18.html#F2

Key points:
*** The change in position of Alpha Centauri AB system in 2026 is approximately 7.400000 arcseconds.
Converted to light-years, this change in separation is approximately 0.000157 light-years.






***alf Cen A sky position: 14 39 36.4940 -60 50 02.373             1
Alpha Cen A: V Magnitude: 0.009999999776482582
Alpha Cen A: Spectral Type: G2V
Alpha Cen A Radial Velocity: -15.252 km/s

***alf Cen B sky position:14 39 35.0631 -60 50 15.099       1
Alpha Cen B: V Magnitude: 1.3300000429153442
Alpha Cen B: Spectral Type: K1V
Alpha Cen B:Radial Velocity: -22.586 km/s

